### ISSUE TITLE
[1.17.1_HD_U_G9_pre29] Glowing potion effect does not work when any shaderpack is enabled (including internal)

### ISSUE THREAD

**Issue by TBlazeWarriorT** [sp614x/optifine#6039]
## Description of Issue
Glowing Effect is not working with any shaderpacks, including internal, tested with Fast Render on and off.

Sometimes invisibility also glitches out and keeps you invisible *even when you don't have the status effect on*. You can see on the screenshot, the effect was removed via commands(look at the chat), was no longer active (check top left), but my player was still fully invisible until the effect timer that used to exist ended (apparently). Does not seem to happen 100% of the time, might have been a one-off bug.
![image](https://user-images.githubusercontent.com/25572454/125875247-c9f25dde-bfcc-46fe-ae91-7fbc86dd3017.png)

## Steps to Reproduce
- Launch mc with pre29
- Set shaders to internal (or any other)
- Give any entity the Glowing Effect
- It will not work

## OptiFine Version
1.17.1_HD_U_G9_pre29

## Installation Method
Standalone

## Log Files/Crash Reports
Nothing relevant to upload

## F3 Debug Screenshot
![image](https://user-images.githubusercontent.com/25572454/125875130-0861f00d-851a-4d4f-aef8-3a9047c3c5c3.png)

## Prior Testing
<!-- If you've installed standalone:
- Does this happen in Vanilla without OptiFine?
No, it's caused by Optifine as usual

## Additional Information
Glowing has been mostly broken for a long time and never got fixed, but this time it broke even harder. Please fix, thanks.

**Comment by fayer3** [comment]
shaders need to implement their own glowing effect,
there already a request to render the glowing effect with vanilla here: #5759

**Comment by TBlazeWarriorT** [comment]
> 
> 
> shaders need to implement their own glowing effect,
> there already a request to render the glowing effect with vanilla here: #5759

Complementary implements the glowing effect and optifine broke even that. I really mean it's not working.
If you mean it being Also broken on Internal shaders is a feature, I call that a very weird feature but ok.
It was sort of working in 1.16.5, the behavior has changed.

**Comment by fayer3** [comment]
Ok, I'm sorry.
didn't get the info from the description, that it doesn't even work in shaders, which should support it.

I tested it with complementary and can confirm that it is broken with 1.17

**Comment by TBlazeWarriorT** [comment]
> 
> 
> Ok, I'm sorry.
> didn't get the info from the description, that it doesn't even work in shaders, which should support it.
> 
> I tested it with complementary and can confirm that it is broken with 1.17

yeah, when I said "any shaderpack" on the very title of this bug report, I really meant any shader. (unless anyone finds a working shader, but I didn't)

**Comment by dbrighthd** [comment]
Im still getting this issue with the full release
